python/ray/tune/trainable.py

```diff
@@ -214,10 +214,20 @@ def save(self, checkpoint_dir=None):
             Checkpoint path that may be passed to restore().
         """
 
-        checkpoint_path = self._save(checkpoint_dir or self.logdir)
+        checkpoint_path = checkpoint_dir or self.logdir
```

Comment:
The checkpoint path needs to include the current iteration right? Otherwise they will collide?

Reply:
done